Simplifying value validation example (#1547)
Roblox/creator-docs: content/en-us/scripting/security/client-server-boundary.md

> In addition to validating types and data, you should validate the values passed through `Class.RemoteEvent|RemoteEvents` and `Class.RemoteFunction|RemoteFunctions`, ensuring they are valid and logical in the context being requested. This is essential for remotes that handle currency, items, numeric inputs, or user-generated content. You must ensure values are within expected boundaries (e.g., a purchase quantity is greater than zero) and that provided IDs or names are valid (e.g., an itemId corresponds to a real item in your game).

Reviewer: ```suggestion
In addition to validating types and data, you should validate the values passed through `Class.RemoteEvent|RemoteEvents` and `Class.RemoteFunction|RemoteFunctions`, ensuring they are valid and logical in the context being requested. This is essential for remotes that handle currency, items, numeric inputs, or user-generated content. You must ensure values are within expected boundaries, for example a purchase quantity being greater than zero or an item ID corresponding to a real item in your game.
```

> **Special considerations for numeric values:** Both inf and NaN are valid number types, but both can cause major issues if an exploiter sends them and they're not handled correctly. Use `Library.math.isfinite()` to detect and reject them:

Reviewer: ```suggestion
Note that both `inf` and NaN are valid number types, but both can cause major issues if an exploiter sends them and they're not handled correctly. Use `Library.math.isfinite()` to detect and reject them:
```
